src/components/MoviesList/MoviesList.tsx

```diff
@@ -1,6 +1,17 @@
 import React from 'react';
+import { MovieCard } from '../MovieCard';
 import './MoviesList.scss';
+import { Movie } from '../../Movie/Movie';
+
+type Props = {
+  movies: Movie[];
+};
+
+export const MoviesList: React.FC<Props> = ({ movies }) => (
+  <div className="movies">
+    {movies && movies.map(movie => (
+      <MovieCard {...movie} key={movie.imdbId} />
```

Comment:
Get rid of spread operator here.
https://mateacademy.slack.com/archives/C02CSGP1KD1/p1636094929188700